src/Management/src/EndpointCore/ActuatorServiceCollectionExtensions.cs

```diff
@@ -11,10 +27,46 @@ public static class ActuatorServiceCollectionExtensions
         public static void RegisterEndpointOptions(this IServiceCollection services, IEndpointOptions options)
         {
             var mgmtOptions = services.BuildServiceProvider().GetServices<IManagementOptions>();
-            foreach (var mgmt in mgmtOptions)
+            foreach (var mgmtOption in mgmtOptions)
             {
-                mgmt.EndpointOptions.Add(options);
+                mgmtOption.EndpointOptions.Add(options);
             }
         }
+
+        public static void AddAllActuators(this IServiceCollection services, IConfiguration config, MediaTypeVersion version)
+        {
+            if (services == null)
+            {
+                throw new ArgumentNullException(nameof(services));
+            }
+
+            if (config == null)
+            {
+                throw new ArgumentNullException(nameof(config));
+            }
+
+            services.AddHypermediaActuator(config);
+
+            if (Environment.OSVersion.Platform == PlatformID.Win32NT)
```

Comment:
```suggestion
            if (Platform.IsWindows)
```

Reply:
refactored to use Platform.IsWindows everywhere